zenodo/modules/pages/templates/zenodo_pages/contact_form.html

```diff
@@ -93,11 +218,96 @@
     var dropdown = document.getElementById('issue_category');
     var current_value = dropdown.options[dropdown.selectedIndex].value;
 
-    for(each in data){
-      if(data[each][0]==current_value){
-        document.getElementById('issue_text').innerHTML = data[each][1];
+    for(each in data)
```

Comment:
You can do `dropdown.options[dropdown.selectedindex].innerHTML = data[current_value]` here. 

Also as a side note, try to generally use intuitive and context-aware variable names inside your `for` loops (eg. `each in data` -> `description in descriptions`).